.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "window.title": "${dirty}${activeEditorShort}${separator}${rootName}${separator}${appName} ${separator} [Branch: features/service-registry-version]"
+}
```

Comment:
This whole file should not be committed. The proper way would be to add `.vscode` to the .gitignore file.

There is one exception though, when we are using dev containers with VScode remote development. In that case the configuration for the debugger should be committed as they rely on some ports specifications which must be obeyed by every user.